presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestEliminateSorts.java

```diff
@@ -87,16 +87,37 @@ public void testNotEliminateSorts()
 
     public void assertUnitPlan(@Language("SQL") String sql, PlanMatchPattern pattern)
     {
-        List<PlanOptimizer> optimizers = ImmutableList.of(
-                new UnaliasSymbolReferences(),
+        List<PlanOptimizer> optimizeBeforeTranslation = ImmutableList.of(
```

Comment:
optimize*r*?

Reply:
Here, we wanted to test two scenarios:
1) `PruneUnreferencedOutputs`/ `UnaliasSymbolReferences` happens after translateExpressions.
2)  `PruneUnreferencedOutputs`/ `UnaliasSymbolReferences` happens before translateExpressions (it should still work).

Reply:
I mean they should be named `optimizersBeforeTranslation` and `optimizersAfterTranslation`